ui/webui/test/check-progress

```diff
@@ -15,18 +15,13 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program; If not, see <http://www.gnu.org/licenses/>.
 
-import os
-import sys
-
 # import Cockpit's machinery for test VMs and its browser test API
-TEST_DIR = os.path.dirname(__file__)
-sys.path.append(os.path.join(TEST_DIR, "helpers"))
-sys.path.append(os.path.join(TEST_DIR, "common"))
-sys.path.append(os.path.join(os.path.dirname(TEST_DIR), "bots/machine"))
+from .parent import setup_paths
+setup_paths()
 
-from installer import Installer
-from testlib import MachineCase, test_main  # pylint: disable=import-error
-from machine_install import VirtInstallMachine
+from .helpers.installer import Installer
+from common.testlib import MachineCase, test_main  # pylint: disable=import-error
```

Comment:
It would be great if all imports of our or Cockpit helper libraries were only relative, or absolute.